TUnit.Assertions/Assertions/Generics/GenericSatisfiesExtensions.cs

```diff
@@ -26,4 +26,39 @@ public static InvokableValueAssertionBuilder<TActual> Satisfies<TActual, TExpect
         return valueSource.RegisterAssertion(new SatisfiesAssertCondition<TActual, TExpected>(asyncMapper, assert, mapperExpression, assertionBuilderExpression),
             [mapperExpression, assertionBuilderExpression]);
     }
+    
+    public static InvokableValueAssertionBuilder<TActual> AllSatisfy<TActual, TInner, TExpected>(
```

Comment:
I think this might read better:

```suggestion
    public static InvokableValueAssertionBuilder<TActual> SatisfiesAll<TActual, TInner, TExpected>(
```

Reply:
I thought that might imply that a single item might satisfy multiple conditions, while the method checks all items if the satisfy a single assertion chain?

Reply:
Ah yeah, that makes sense.